.gitpod.yml

```diff
@@ -0,0 +1,15 @@
+image:
+  file: .gitpod.Dockerfile
+
+ports:
+  # Sidecar API port
+  - port: 3999
+    onOpen: open-preview
+  - port: 1000-65535
+    onOpen: ignore
+
+tasks:
+  - init: npm install
```

Comment:
it took me 3+ minutes to get install done. is there a way to get this done on the docker level? So it would happen only once and everyone opening the pod will be all set?

Reply:
We can enable gitpod integration into the repo for "prebuilt workspace" https://www.gitpod.io/docs/prebuilds/

Reply:
I just enabled the prebuild feature. Kinda funky but seems faster

Reply:
(5 hours later) clicked the gitpod link and it wasn't using a prebuilt workspace like it did initially, maybe it doesn't stay cached unless on master? No idea, not much documentation on their end.

Reply:
Let's just move ahead for now. Can look into that when we see it used more and someone complaining about it :)